trustymail/domain.py

```diff
@@ -51,7 +51,7 @@ def format_list(record_list):
     # anything else.  Thus this if clause handles only empty
     # lists.  This makes a "null" appear in the JSON output for
     # empty lists, as expected.
-    if not record_list:
+    if not record_list or len(record_list) == 0:
```

Comment:
This code change does not modify the logic.  It will not be executed as a list of length `0` will already evaluate to `false` and short circuit the `or`.

Reply:
Best meta joke about DNSSEC I've seen today

:wave: :eyes: @echudow

Reply:
🤣

Reply:
Reverted in 135833d.

Reply:
Sorry, I'm too used to other languages where that check would have been useful.  
@h-m-f-t, I'm sure you'll find some other jokes about DNSSEC to throw my way too.